backend/api/user.go

```diff
@@ -105,11 +170,12 @@ func ModifyInfo(c *gin.Context) {
 
 	if modPassword != "" {
 		if err := pkg.Compare(user.Password, modPassword); err == nil {
-			c.JSON(http.StatusOK, gin.H{"result": "original password is the same as the modified one"})
+			c.JSON(http.StatusBadRequest, gin.H{"result": "original & modified passwords are the same"})
```

Comment:
I think it is acceptable to use the same password

Reply:
I assume the user wants to change his/her password, as it has been exposed, for instance. Therefore, we should remind the user if the modified password is the same. If the user does not want to change the password, then **modPassword=""** should be passed.

Reply:
Okay